### PR TITLE
Output none label fix and changed contact to author

### LINKED ISSUE #93: Rename contact in OPAT schema to author
The "contact" field is ambiguous and we will add vendor in #91 so change it to "author".

### PR DESCRIPTION
**Pull request by dmundra**
Fixes #93 and a bug where the 'none' label was still appearing with an empty colon in the output.

### CODE REVIEW

File: tests/examples/valid.markdown [hunk 1]
```diff
@@ -6,12 +6,12 @@ Based on VPAT® 2.4 edition 508/WCAG 2.0
...
 
 ## Report Date
-8/4/2021
+8/5/2021
 
 ## Product Description
 Content Management System
 
-## Contact Information
+## Author Information
```
Comment (mgifford) [inline tests/examples/valid.markdown:14]: I'm tempted to leave this as Contact Information. In this case we are both the Vendor and the Author and we will want to have both ### Vendor & ### Author might be one way to resolve this?

**Review by mgifford — COMMENTED** [review]
All looks good. My only hesitation is listed in the comment below. Happy to approve (or for you to) when we've got a solution there.

File: tests/examples/valid.markdown (hunk 1 above)
Comment (dmundra) [reply tests/examples/valid.markdown:14]: Good point. I can make that change.

Comment (dmundra) [reply tests/examples/valid.markdown]: @mgifford updated.

**Review by mgifford — APPROVED** [review]
Looks good to go. Thanks Daniel.

**Pull request merged**